pages/common/ccache.md

```diff
@@ -0,0 +1,24 @@
+# ccache
+
+> C/C++ compiler cache.
+> More information: <https://ccache.dev/manual/4.9.html#_command_line_options>.
+
+- Show current cache [s]tatistics:
+
+`ccache -s`
+
+- Clear all [C]ache:
+
+`ccache -C`
+
+- Reset ([z]ero) statistics (but not cache itself):
+
+`ccache -z`
+
+- Compile a file and cache compiled output:
+
+`ccache gcc main.c`
+
+- Create a symlink to ccache to automatically act as the given compiler (note: some symlinked files may already exist in `/usr/lib/ccache/bin/`):
+
+`ln -s /usr/bin/ccache gcc && ./gcc main.c`
```

Comment:
In my opinion, this example should be removed. Someone can have `sccache` installed in a different path, which will result in an error.

We usually only document usage of the command itself.
```suggestion
```

Reply:
This documentation is about `ccache`, not `sccache` tho

The point of this example is to avoid making users think `ccache gcc main.c` is the best way to use ccache, by showing them the how ccache can directly act as the given compiler

I personally think this example should stay, but I'll remove it if you want to

Reply:
> This documentation is about `ccache`, not `sccache` though

That was a typo, sorry.

All symlinks are by default in `/usr/lib/ccache/bin`. Prepending `/usr/lib/ccache/bin` to your `$PATH` allows you to use them without providing an absolute path, whereas this example creates new symlinks in the current working directory, which isn't very useful in my opinion.